README.md

```diff
@@ -8,312 +8,297 @@ United States Government Sponsorship acknowledged. Any commercial use must be ne
 This software may be subject to U.S. export control laws. By accepting this software, the user agrees to comply with all applicable U.S. export laws and regulations. User has the responsibility to obtain export licenses, or other export authority as may be required before exporting such information to foreign countries or providing access to foreign persons.
 
 # TimeCraftJS
+
 TimeCraftJS is a time conversion library that uses [NAIF's CSPICE](https://naif.jpl.nasa.gov/naif/).
 
-It is of extreme importance for different systems to understand time and calculate operations consistently. TimeCraftJS exposes functions to execute operations to convert SCET, ET, SCT, furnish Kernels, among others, in the client side. This avoid unnecessary trips to the backend and makes allows for important time operations to happen if network connectivity is not immediately available.  
+It is of extreme importance for different systems to understand time and calculate operations consistently. TimeCraftJS exposes functions to execute operations to convert SCET, ET, SCT, furnish Kernels, among others, in the client side. This avoid unnecessary trips to the backend and makes allows for important time operations to happen if network connectivity is not immediately available.
 
-## Quick Start
+Some basic kernels are provided in this repo for performing time conversions. More kernels for other missions can be found [here](https://naif.jpl.nasa.gov/pub/naif/pds/data/).
 
-### Install via NPM
+## Quick Start
 
-1. Install TimeCraftJS via NPM:
+### Installation via NPM
 
 ```shell
-npm install --savedev timecraftjs
+npm install timecraftjs
 ```
 
-2. Import the libraries:
+### Use with Webpack
 
-```html
-<script type="text/javascript" src="./node_modules/spice.js"></script>
-<script type="text/javascript" src="./node_modules/timecraft.js"></script>
-<script type="text/javascript" src="./node_modules/cspice.js"></script>
-```
+The compiled emscripten blob includes `require( 'fs' )` which can cause Webpack to fail to compile for the web. In this case add the following options to the root of your Webpack config:
 
-3. Start calling the timecraft functions:
+```js
+node: {
 
-```javascript
-window.addEventListener('timecraftready',function(){
-		console.log(timecraft.bodc2n(499));
-		console.log(timecraft.et2utc(43523178.23,"ISOC",4));
-	});
+    fs: 'empty'
+    
+}
 ```
 
-### Browser testing
+### Time Conversion
 
-1.  Clone the Repository
-```shell
-git clone https://github.com/NASA-AMMOS/timecraftjs.git
+```js
+import * as TimeCraft from 'timecraftjs';
+
+// Load the kernels
+const kernelBuffers = await Promise.all( [
+    
```

Comment:
You certainly love your white-space :)

Reply:
Heh habit from three.js now. I do think the whitespace makes it easier to read which is good for docs but if you'd prefer I can remove it.